Networking/Networking/Requests/AuthenticatedRequest.swift

```diff
@@ -18,9 +21,13 @@ struct AuthenticatedRequest: URLRequestConvertible {
     /// Returns the Wrapped Request, but with a WordPress.com Bearer Token set up.
     ///
     func asURLRequest() throws -> URLRequest {
+        guard case let .wpcom(username: _, authToken: authToken, siteAddress: _) = credentials else {
```

Comment:
Nit: This can be simplified as:
```suggestion
        guard case let .wpcom(_, authToken, _) = credentials else {
```